app/appmessage/rpc_get_coin_supply.go

```diff
@@ -34,7 +34,7 @@ func (msg *GetCoinSupplyResponseMessage) Command() MessageCommand {
 // NewGetCoinSupplyResponseMessage returns a instance of the message
 func NewGetCoinSupplyResponseMessage(totalSompi uint64, circulatingSompi uint64) *GetCoinSupplyResponseMessage {
```

Comment:
Please call it maxSompi in here as well

Reply:
i cannot find this one in my code. i think it was changed by commit #72dcf87 but you reveiwed #ab1e7dc